ssl-impl/src/main/java/io/crate/protocols/ssl/SslConfiguration.java

```diff
@@ -84,7 +85,7 @@ static SslContext buildSslContext(Settings settings) {
                 trustManagers = trustStoreSettings.get().trustManagers;
             }
 
-            SSLContext sslContext = SSLContext.getInstance("TLS");
+            SSLContext sslContext = SSLContext.getInstance("TLSv1.2");
```

Comment:
This is the newest standard of TLS. If you get "TLS", the TLS version depends on the JVM you're using. I wanted to be explicit here. Will add a comment.